certbot/tests/cli_test.py

```diff
@@ -1,4 +1,7 @@
 """Tests for certbot.cli."""
+# Many tests in this file should be moved
+# into main_test.py and renewal_test.py.
```

Comment:
Ah I see. I noticed this problem while working on the tests for account deactivation.